admin_guide/upgrading_citus.rst

```diff
@@ -76,7 +76,15 @@ After installing the new package and restarting the database, run the extension
 
 .. note::
 
-  During a major version upgrade, from the moment of yum installing a new version, Citus will refuse to run distributed queries until is restarted and ALTER EXTENSION is executed. This is to protect your data, as Citus object and function definitions are specific to a version. After a yum install you should (a) restart and (b) run alter extension.
+  During a major version upgrade, from the moment of yum installing a new
+  version, Citus will refuse to run distributed queries until is restarted and
```

Comment:
Just noticed that this PR preserves one of my own typos: "queries until is restarted." Want to fix it up while you're at it?